microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
```

Comment:
I know that Tomasz is the lead person, but shouldn't we user our names when we code parts. Thus, it would be easier to find the right person for upcoming questions.